src/qmapshack/dem/IDem.cpp

```diff
@@ -184,6 +188,11 @@ void IDem::slotSetFactorHillshade(int f)
     }
 }
 
+void IDem::slotSetFactorSlopeShade(int f)
+{
+    factorSlopeShading = f / 100.;
```

Comment:
what is this factor shift good for?

Reply:
Qt sliders work with integer values. Therefore mapping slope shading slider's integer value range [25 .. 300] to slope shading's scaling range [0.25 .. 3.0] is required.
Analogue to hill shading slider's value mapping factor _factorHillshading_.